semantics/cpp/language/translation/decl/class.k

```diff
@@ -546,8 +546,10 @@ syntax Stmt ::= Defaulted(SpecialMemberFunction, Class) [klabel(DefaultedSMF)]
           <initializers>... X |-> (T |-> I::Init) ...</initializers>
           requires isDeletedDefaultConstructor(X, T, C, FuncT, VM, I)
 
+     syntax KItem ::= "dotInit"
```

Comment:
Why we cannot have `.Init`?

Reply:
Good suggestion, it just didn't occur to me. I'll change it.